roles/minimalbackup/tasks/mysql_variables.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Print MySQL / MariaDB version
+  copy:
+    content: "{{ minimalbackup_mysql_info.version.full }}"
+    dest: "{{ minimalbackup_tmpdir }}/{{ ansible_hostname }}_mysql_version"
+  delegate_to: localhost
+
+- name: Fetch MySQL / MariaDB variables
+  community.mysql.mysql_query:
+    query: "SHOW VARIABLES"
+  register: minimalbackup_mysql_variables
+
+- name: Print MySQL / MariaDB variables
```

Comment:
The print of the Mysql variables may not be idempotent in some cases. However, this is of course to be expected.
So that the pipelines do not fail here, we should probably switch off the idempotent lint at this point.

Reply:
Good catch. Just did that.